src/worker/environment.rs

```diff
@@ -30,6 +31,7 @@ pub struct Environment {
     pub deadpool: Pool<AsyncPgConnection>,
     pub emails: Emails,
     pub team_repo: Box<dyn TeamRepo + Send + Sync>,
+    pub docs_rs: Arc<dyn DocsRsClient>,
```

Comment:
I guess it might be worth making this an `Option` in case the API token is not set via env var.